com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -101,6 +101,7 @@ internal void QueueAssetLoading()
                     refVol.dilationValidtyThreshold = refVol.sceneData.GetBakeSettingsForScene(gameObject.scene).dilationSettings.dilationValidityThreshold;
                 }
 #endif
+                m_PreviousState = m_CurrentState;
```

Comment:
Note for @JulienIgnace-Unity : this is the change we discussed the other day for the double-loading, since we now spawn an error on wrong loading, having it spawn twice is annoying :D